### ISSUE TITLE
Broken syntax highlighting for the language VHDL

### ISSUE THREAD

**Issue by Paebbels** [github/linguist#3750]
I reported a broken syntax highlighting on GitHub to the GitHub support. They redirected me to this project.

> Hi Patrick,
>
> Thanks for reaching out!
>
> We use open source TextMate-style language grammars for syntax highlighting, which are available here: https://github.com/github/linguist/blob/master/grammars.yml
>
> The fastest way to get this fixed would be to open an issue or contribute a fix for your specific grammar concern in the repository.
>
> Linguist pulls in grammar updates with each new release, which usually happens every couple of weeks. This should get fixed on GitHub.com shortly after that happens.
>
>I hope that helps! Let us know if there's anything else we can do.
>
>Regards,
>Friday
>
>> Hello GitHub's syntax highlighting has some bugs. Please see this issue for the broken highlighting and also an in detail explanation for the underlying problem by one of the users. (End of the issue).
>> 
>> [Issue 395 at tgingold/ghdl](https://github.com/tgingold/ghdl/issues/395)

**Comment by Alhadis** [comment]
Hopefully you can tolerate another redirect then, because Linguist isn't responsible for generating and displaying the syntax highlighting you see on the site. ;-) The URL that GitHub support should have redirected you to is [this one, here](https://github.com/github/linguist/tree/master/vendor).

If you search for `VHDL` on that page, you'll see a link to the responsible grammar repository: [`textmate/vhdl.tmbundle`](https://github.com/textmate/vhdl.tmbundle). Linguist's job is to select the grammar package to provide the highlighting for a specific language... the final result is handled by the selected grammar.

In short, file an issue at [`textmate/vhdl.tmbundle`](https://github.com/textmate/vhdl.tmbundle), not here. =)
